src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -915,8 +914,15 @@ export default class EthereumApi implements Api {
     const parentHeader = parentBlock.header;
     const options = this.#options;
 
-    const generateVM = () => {
-      return blockchain.vm.copy();
+    const generateVM = async () => {
+      // note(hack): blockchain.vm.copy() doesn't work so we just do it this way
```

Comment:
I just noticed this hack that has slipped through, though I don't know if I'd call it a hack... More a just a hit against performance for gas estimation. Thoughts on removing comment and/or making an issue to look into it at a later date?
@davidmurdoch @tenthirtyone @jeffsmale90

Reply:
Let's add a new issue for it

Reply:
Done, #3980